screens/authentication/signUp/SignUpScreen.jsx

```diff
@@ -3,14 +3,13 @@ import PropTypes from 'prop-types';
 import {
   Text,
   View,
-  SafeAreaView,
-  TextInput,
   TouchableOpacity,
   KeyboardAvoidingView,
   TouchableWithoutFeedback,
   Keyboard,
   Platform,
   Alert,
+  ScrollView,
 } from 'react-native';
 import {Formik, useFormik} from 'formik';
 import {signUpValidationSchema} from './signUpValidationSchema';
```

Comment:
you don't have signUpValidationSchema file ??

Reply:
is in the same folder